src/include/distributed/comment.h

```diff
@@ -0,0 +1,31 @@
+/*-------------------------------------------------------------------------
+ *
+ * comment.h
+ *    Declarations for comment related operations.
+ *
+ * Copyright (c) Citus Data, Inc.
+ *
+ *-------------------------------------------------------------------------
+ */
+
+#ifndef COMMENT_H
+#define COMMENT_H
+
+#include "postgres.h"
+
+#include "nodes/parsenodes.h"
+
+typedef struct
+{
+	char *name;
+	int type;
```

Comment:
This field is now unused

Reply:
It is being used as the data type of ObjectTypeInfos

Reply:
Sure, but nowhere do we actually read from the `type` field. Since `IsA` change it's an unused field.

Reply:
Got it. Thanks changed now